src/js/index.js

```diff
@@ -0,0 +1,218 @@
+import CreatePage from './components/createHtml/createHtml';
+import * as board from './components/main/board';
+
+import '../css/main.scss';
+
+const createPage = new CreatePage();
+
+const LANG = { RU: 'ru', EN: 'en' };
+
+const state = {
+  lang: LANG.RU,
+  capsLock: false,
+};
+
+const KEYBOARD = {
+  tab: '  ',
+  space: ' ',
+  enter: '\n',
+};
+
+window.addEventListener('beforeunload', () => {
+  localStorage.setItem('language', state.lang);
+});
+
+window.addEventListener('DOMContentLoaded', () => {
+  const language = localStorage.getItem('language');
+  if (language !== null) {
+    state.lang = language;
+  }
+});
+
+window.addEventListener('DOMContentLoaded', () => {
+  createPage.create();
+  board.createKeyBoard(state.lang);
+
+  const keyBoard = document.querySelector('.keyboard-list');
+
+  document.addEventListener('keydown', (event) => {
+    const switcherCtrlAlt = event.ctrlKey && event.altKey;
+    const switcherShiftAlt = event.shiftKey && event.altKey;
+    if (switcherCtrlAlt || switcherShiftAlt) {
+      state.lang = state.lang === LANG.RU ? LANG.EN : LANG.RU;
+      keyBoard.innerHTML = '';
+      keyBoard.innerHTML = board.generateButtons(state.lang);
+    }
+  });
+
+  const textarea = document.querySelector('.textarea');
+  const capsLock = document.querySelector('.capslock');
+
+  const capsLockHandler = () => {
+    const buttons = document.querySelectorAll('.keyboard__button');
+    if (!state.capsLock) {
+      buttons.forEach((button) => {
+        const currentButtons = button;
+        if (button.textContent.trim().length === 1) {
+          state.capsLock = true;
+          currentButtons.textContent = button.textContent.toUpperCase();
+          currentButtons.dataset.keyCode = button.textContent.toUpperCase();
+          capsLock.classList.add('keyboard__button_active');
+        }
+      });
+    } else {
+      buttons.forEach((button) => {
+        const currentButtons = button;
+        if (currentButtons.textContent.trim().length === 1) {
+          state.capsLock = false;
+          currentButtons.textContent = button.textContent.toLowerCase();
+          currentButtons.dataset.keyCode = button.textContent.toLowerCase();
+          capsLock.classList.remove('keyboard__button_active');
+        }
+      });
+    }
+  };
+
+  const shiftHandlerUp = () => {
+    const buttons = document.querySelectorAll('.keyboard__button');
+    buttons.forEach((button) => {
+      const currentButtons = button;
+      if (button.textContent.trim().length === 1) {
```

Comment:
Same here, use RegExp to find letter buttons.

Reply:
made letterButtons function